api/src/pcapi/routes/backoffice/templates/offer/list.html

```diff
@@ -189,6 +190,11 @@
           {% endif %}
         {% endif %}
       </td>
+      <td>
+        {% if row.offer_min_booking_limit_date %}
+          {{ row.offer_min_booking_limit_date | format_date("%d/%m/%Y") }}
+        {% endif %}
```

Comment:
La date vide est normalement déjà gérée dans le filtre `format_date`, on devrait donc pouvoir simplifier : 

```suggestion
        {{ row.offer_min_booking_limit_date | format_date("%d/%m/%Y") }}
```